app/views/projects/_project.html.erb

```diff
@@ -34,9 +34,11 @@
 
       <div class="col-md-12 members">
         <ul class="member-list">
-          <% project.users.first(Project::MAX_MEMBERS_PER_CARD).each do |member| %>
+          <% project.users.first(Project::MAX_MEMBERS_PER_CARD).each do |user| %>
             <li class="member">
-              <img src="data:image/png;base64,<%= RubyIdenticon.create_base64(member.username) %>" class="identicon" data-toggle="tooltip" data-placement="top" title="<%= member.name %>">
+              <% cache user.username do %>
```

Comment:
@akitaonrails maybe `user/:user_email/icon` then, what do you think? If the user change its email the avatar should change too.